src/OpenLoco/src/Windows/Cheats.cpp

```diff
@@ -508,7 +508,7 @@ namespace OpenLoco::Ui::Windows::Cheats
         static constexpr auto _widgets = makeWidgets(
             Common::makeCommonWidgets(kWindowSize.width, kWindowSize.height, StringIds::company_cheats),
             makeWidget({ 4, 48 }, { kWindowSize.width - 8, 33 }, WidgetType::groupbox, WindowColour::secondary, StringIds::cheat_select_target_company),
-            makeDropdownWidgets({ 10, 62 }, { kWindowSize.width - 20, 12 }, WidgetType::textbox, WindowColour::secondary),
+            makeDropdownWidget({ 10, 62 }, { kWindowSize.width - 20, 12 }, WindowColour::secondary),
```

Comment:
While this is clearly better, I think this might have broken the mouseDown event. It's triggering on `Widx::target_company_dropdown`, but should now be triggering on `Widx::target_company_dropdown_btn`:

```diff
         static void onMouseDown(Window& self, WidgetIndex_t widgetIndex)
         {
-            if (widgetIndex == Widx::target_company_dropdown)
-                Dropdown::populateCompanySelect(&self, &self.widgets[widgetIndex]);
+            if (widgetIndex == Widx::target_company_dropdown_btn)
+                Dropdown::populateCompanySelect(&self, &self.widgets[widgetIndex - 1]);
```

Reply:
Oh yea.. I'll address that.